mne/channels/layout.py

```diff
@@ -777,9 +780,16 @@ def read_montage(kind, ch_names=None, path=None, scale=True):
     path : str | None
         The path of the folder containing the montage file. Defaults to the
         mne/channels/data/montages folder in your mne-python installation.
-    scale : bool
-        Apply useful scaling for out the box plotting using montage.pos.
-        Defaults to True.
+    unit : 'mm' | 'm' | None
+        Values supported are 'mm', which will scale points up by 1e3,
+        'm' which will scale point down by 1e3, or None.
+        Default is None.
```

Comment:
This seems a little more complicated to me than it has to be. What use cases are there for this? The only use case I can think of is where your input montage is in mm but you want the sensors in MNE standard which is m. For this use case I would have a parameter `unit='mm'`, or `input_unit='mm'`. The default should be `unit='m'` or equivalent, in which case no scaling is necessary.